ui/desktop/app/styles/app.scss

```diff
@@ -19,6 +19,25 @@
   }
 }
 
+/**
+ * This section enables scrolling within the sidebar columns separately,
+ * rather than the whole viewport.  This ensures the sidebar and application
+ * header remain visible when scrolling.
+ */
+.rose-layout-global {
```

Comment:
Is this a section I deleted from app.scss in initial days of desktop client when we didn't have target/sessions views? If so, sorry!